### PR TITLE
Display current and target scores for benchmark indicators

### PR DESCRIPTION
**Pull request by dysnomian**
Exports scores and goals for each benchmark indicator as part of the state from server json, plumbs it through to the Indicator components, and shows it there. Includes a scss update.

Unsure how best to test this.

### CODE REVIEW

File: app/models/plan.rb [hunk 1]
```diff
@@ -152,14 +152,15 @@ def excluded_actions_for(benchmark_indicator)
   def current_and_target_scores
     indicators = BenchmarkIndicator.all.pluck(:id)
 
-    indicators.reduce({}) do |sag, i|
-      score = score_value_for(assessment_indicator: i)
+    indicators.reduce({}) do |scores, indicator_id|
+      current = score_value_for(assessment_indicator: indicator_id)
```
Comment (indirect) [inline app/models/plan.rb:156]: unless I'm missing something, isn't the method `score_value_for` only present on `PlanBuilder`?

Comment (indirect) [reply app/models/plan.rb:156]: also, isn't this `indicator_id` a `BenchmarkIndicator` id, and not an `AssessmentIndicator` id? that's probably not going to return the score that you are looking for if so :)

Comment (dysnomian) [reply app/models/plan.rb]: > unless I'm missing something, isn't the method `score_value_for` only present on `PlanBuilder`?

Yes, but `PlanBuilder` is included in the Plan model

Comment (dysnomian) [reply app/models/plan.rb]: > also, isn't this `indicator_id` a `BenchmarkIndicator` id, and not an `AssessmentIndicator` id? that's probably not going to return the score that you are looking for if so :)

Riiiiight... fixing that

**Review by indirect — APPROVED** [review]

**Pull request merged**